awswrangler/dynamodb/_read.py

```diff
@@ -146,3 +150,331 @@ def read_partiql_query(
+
+    >>> import awswrangler as wr
+    >>> from boto3.dynamodb.conditions import Attr
+    >>> df = wr.dynamodb.read_items(
+    ...     table_name='my-table',
+    ...     filter_expression=Attr('my_attr').eq('this-value')
+    ... )
+
+    Same as above, but with FilterExpression as string
+
+    >>> import awswrangler as wr
+    >>> df = wr.dynamodb.read_items(
+    ...     table_name='my-table',
+    ...     filter_expression='my_attr = :v',
+    ...     expression_attribute_values={':v': 'this-value'}
+    ... )
+
+    Reading items involving an attribute which collides with DynamoDB reserved keywords
+
+    >>> import awswrangler as wr
+    >>> df = wr.dynamodb.read_items(
+    ...     table_name='my-table',
+    ...     filter_expression='#operator = :v',
+    ...     expression_attribute_names={'#operator': 'operator'},
+    ...     expression_attribute_values={':v': 'this-value'}
+    ... )
+
+    """
+    # Extract key schema
+    table_key_schema = get_table(table_name=table_name, boto3_session=boto3_session).key_schema
+
+    # Detect sort key, if any
+    if len(table_key_schema) == 1:
+        partition_key, sort_key = table_key_schema[0]["AttributeName"], None
+    else:
+        partition_key, sort_key = (
+            next(filter(lambda x: x["KeyType"] == "HASH", table_key_schema))["AttributeName"],
+            next(filter(lambda x: x["KeyType"] == "RANGE", table_key_schema))["AttributeName"],
+        )
+
+    # Build kwargs shared by read methods
+    kwargs: Dict[str, Any] = {"ConsistentRead": consistent}
+    if partition_values:
+        if sort_key is None:
+            keys = [{partition_key: pv} for pv in partition_values]
+        else:
+            if not sort_values:
+                raise exceptions.InvalidArgumentType(
+                    f"Kwarg sort_values must be specified: table {table_name} has {sort_key} as sort key."
+                )
+            if len(sort_values) != len(partition_values):
+                raise exceptions.InvalidArgumentCombination("Partition and sort values must have the same length.")
+            keys = [{partition_key: pv, sort_key: sv} for pv, sv in zip(partition_values, sort_values)]
+        kwargs["Keys"] = keys
+    if key_condition_expression:
+        kwargs["KeyConditionExpression"] = key_condition_expression
+    if filter_expression:
+        kwargs["FilterExpression"] = filter_expression
+    if columns:
+        kwargs["ProjectionExpression"] = ", ".join(columns)
+    if expression_attribute_names:
+        kwargs["ExpressionAttributeNames"] = expression_attribute_names
+    if expression_attribute_values:
+        kwargs["ExpressionAttributeValues"] = expression_attribute_values
+    if max_items_evaluated:
+        kwargs["Limit"] = max_items_evaluated
+
+    # If kwargs are sufficiently informative, proceed with actual read op
+    if partition_values or key_condition_expression or filter_expression or allow_full_scan or max_items_evaluated:
```

Comment:
Using `any` would be simpler perhaps?
`if any((partition_values, key_condition_expression,...)):`

Reply:
Definitely, thanks.